Makefile

```diff
@@ -7,3 +7,58 @@ generate_migration:	## Generate a new migration
 
 generate_api_clients:	## Generate API clients
 	go generate ./api-clients/convoy/generate.go
+	
+.PHONY: docker_services firebase_emulator launch_scenario ingest_data send_webhooks cron_scheduler workers reset_db
+
+# Default environment file
+ENV_FILE=.env
+
+## Start docker services
+docker_services:
```

Comment:
I feel this is almost as long as the command it replaces. I don't think `docker compose up` warrants a separate command?